app/build.gradle

```diff
@@ -399,12 +399,13 @@ jar {
     from fileTree(buildDir).matching { include 'META-INF/services/*' }
 }
 
-task collectEnterpriseModules(dependsOn: [':users:jar']) {
+task collectEnterpriseModules(dependsOn: [':users:jar', ':ssl:jar']) {
     // this task collects all enterprise module jars into a single folder
     // named enterprise from which the files are taken for the installDist task
     doLast {
         copy {
             from(project(':users').tasks.jar.archivePath)
+            from(project(':ssl').tasks.jar.archivePath)
```

Comment:
I think the interface itself shouldn't be within an enterprise-only jar. Since sql already has a compile dependency it shouldn't be necessary to include the jar here?